doc/source/user_guide/indexing.rst

```diff
@@ -1475,17 +1475,22 @@ default value.
    s.get('a')  # equivalent to s['a']
    s.get('x', default=-1)
 
-The :meth:`~pandas.DataFrame.lookup` method
+Looking up values by index/column labels
 -------------------------------------------
```

Comment:
Make sure to adjust the heading size; this might cause strange issues in CI

Reply:
Could you elaborate? Im not sure if I should do anything with the size / header right (`#`). If I see correctly, all other headers are defined the same, for example

```
Duplicate data
--------------
```

Reply:
```suggestion
----------------------------------------
```

If reading correctly over the web this is the correct length

Reply:
Yes, thank you, didn't know the amount of dashes mattered. I comitted your change.